lmdeploy/cli/cli.py

```diff
@@ -104,10 +102,7 @@ def add_parser_chat():
             ', "baichuan-inc/baichuan2-7b-chat" and so on')
         # common args
         ArgumentHelper.backend(parser)
-        ArgumentHelper.trust_remote_code(parser)
         # # chat template args
-        ArgumentHelper.meta_instruction(parser)
-        ArgumentHelper.cap(parser)
```

Comment:
should we update the doc. it is  not included in readthedocs though.
https://github.com/InternLM/lmdeploy/blob/main/docs/en/supported_models/codellama.md#completion